src/Lykke.Service.PayInvoice.Repositories/Extensions/Extensions.cs

```diff
@@ -19,6 +20,16 @@ public static string PropertyNotEqual(this string property, string value)
             return TableQuery.GenerateFilterCondition(property, QueryComparisons.NotEqual, value);
         }
 
+        public static string DateGreaterThanOrEqual(this string property, DateTime value)
```

Comment:
Tha same code was added to PayInternal service. Would be better to add it to common library.